packages/javascript-sdk/jest.basic.config.ts

```diff
@@ -7,10 +7,10 @@
  * This software may be modified and distributed under the terms
  * of the MIT license. See the LICENSE file for details.
  */
-const config = require('./jest.config');
+import config from './jest.config';
 
-module.exports = {
-  ...config,
+export default {
+  // ...config,
```

Comment:
If we are no longer spreading `config` into the exported object here, do we need to import it?

Reply:
ah good catch, we need this. i had to comment these out to make the upgrade work, so i have to comment it back in.